client/asset/interface.go

```diff
@@ -42,6 +43,9 @@ type WalletConfig struct {
 	// Account is the name of the wallet account. The account and wallet must
 	// already exist.
 	Account string
+	// FallbackFeeRate is fee rate used for withdraws, refunds, and DEX fee
+	// payment transactions when optimal fee estimation is unavailable.
+	FallbackFeeRate uint64
```

Comment:
Thought about `ConfigOpts`, but I got the impression that was for wallet software (dcrwallet, bitcoind, etc.) settings rather than settings for the Go wallet implementation.  Will revisit this.

Reply:
Anyway, punt as per matrix chat.  @itswisdomagain Maybe this could be a follow up for you.  There's a sort of obsolete `WalletInfo.DefaultFeeRate` now and `WalletConfig.FallbackFeeRate`.  The best way to clean this up depends on when, how, and where we want the fallback fee rate to be set by the user.

Reply:
Right. Will think up and share some ideas and put up a PR once an issue for this is put up.